### PR TITLE
feat: add routeConfig to the stub snapshot and route

### PR DESCRIPTION
**Pull request by jnizet**

### CODE REVIEW

File: projects/ngx-speculoos/src/lib/route.ts [hunk 1]
```diff
@@ -228,6 +233,16 @@ class ActivatedRouteSnapshotStub extends ActivatedRouteSnapshot {
     this._pathFromRoot = value;
   }
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```
Comment (cexbrayat) [inline projects/ngx-speculoos/src/lib/route.ts:237]: Why is this necessary?

**Review by cexbrayat — APPROVED** [review]

File: projects/ngx-speculoos/src/lib/route.ts (hunk 1 above)
Comment (jnizet) [reply projects/ngx-speculoos/src/lib/route.ts:237]: Because TypeScript complains that I override a property using a getter. But since the property is readonly, and there is no documented constructor to pass the value for this readonly member, I don't really see a betterr choice
